mlflow/R/mlflow/R/cli.R

```diff
@@ -40,6 +29,11 @@ mlflow_cli <- function(...,
     MLFLOW_CONDA_HOME = python_conda_home(),
     MLFLOW_TRACKING_URI = mlflow_get_tracking_uri()
   ), env)
+  if (is.null(stderr_callback)) {
+    stderr_callback <- function(x, p) {
+      cat(x, stderr())
```

Comment:
1. Unlike print, cat does not add newline. I am not sure, does the input contain newline characters or should we add one?

2. Don'f you need to do cat(x, file = stderr())? I think the above would either print info about stderr or fail cause cat can not print lists.

Reply:
Ah nice catch you're right about needing the named file arg - without it `cat` will just print the file descriptor for stderr(). For 2),  the CLI commands do generate stderr with newlines. Try for example running `R CMD build . && Rscript -e 'mlflow:::mlflow_cli("server", "--help")'` on this PR branch.